cli/generate-component.js

```diff
@@ -0,0 +1,131 @@
+const fs = require('fs');
+const inquirer = require('inquirer');
+const { stripIndent } = require('common-tags');
+
+const ComponentTypes = {
```

Comment:
That's really a minor thing, but why not use singular for types?
`Boolean`, `DayOfWeek`, `ComponentType` etc. (we're not using e.g. `Booleans` even though it can be regarded as an enum of two values).
Even literals look & read better this way:
`DayOfWeek.FRIDAY` "day of week Friday"
`ComponentType.FUNCTION` "component type "function""
That's Organic English!!